assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/useHeaderRowGenerator.tsx

```diff
@@ -86,9 +86,13 @@ const useHeaderRowGenerator = (): DatesTableHeaderRowGen => {
 			{
 				key: 'actions',
 				type: 'cell',
-				className:
-					'ee-date-list-col-hdr ee-date-list-col-actions ee-rspnsv-table-column-big ee-centered-column',
-				value: <span className={'ee-rspnsv-table-long-label'}>{__('Actions')}</span>,
+				className: 'ee-date-list-col-hdr ee-actions-column ee-rspnsv-table-column-big ee-centered-column',
+				value: (
+					<>
+						<span className={'ee-rspnsv-table-long-label'}>{__('Actions')}</span>
+						<span className={'ee-rspnsv-table-short-label'}>{__('Actions')}</span>
```

Comment:
as seen in original issue, column header wasn't visible:

![image](https://user-images.githubusercontent.com/1477453/82046780-3bd32c00-96ba-11ea-9c31-b827deb830a2.png)